additional-labs/resources/job_mysql-dump.yaml

```diff
@@ -24,20 +24,20 @@ spec:
               valueFrom:
                 secretKeyRef:
                   key: database-name
-                  name: mysql
+                  name: mariadb
             - name: MYSQL_USER
               valueFrom:
                 secretKeyRef:
                   key: database-user
-                  name: mysql
+                  name: mariadb
             - name: MYSQL_HOST
-              value: mysql
+              value: mariadb
             - name: MYSQL_PORT
               value: "3306"
             - name: MYSQL_PASSWORD
               valueFrom:
                 secretKeyRef:
                   key: database-password
-                  name: mysql
+                  name: mariadb

```

Comment:
If we change everything from mysql to mariadb, should the files also be renamed?